code-examples/quarkus-kafka-registry-quickstart/src/main/resources/application.properties

```diff
@@ -0,0 +1,50 @@
+# Quarkus config
+quarkus.ssl.native=true
+
+# Configure the Kafka sink (we write to it)
+mp.messaging.outgoing.generated-price.connector=smallrye-kafka
+mp.messaging.outgoing.generated-price.topic=prices-complex
+mp.messaging.outgoing.generated-price.value.serializer=io.apicurio.registry.serde.avro.AvroKafkaSerializer
+mp.messaging.outgoing.generated-price.apicurio.registry.url=${REGISTRY_URL}
+mp.messaging.outgoing.generated-price.apicurio.registry.auto-register=true
+mp.messaging.outgoing.generated-price.apicurio.registry.avro-datum-provider=io.apicurio.registry.serde.avro.ReflectAvroDatumProvider
+mp.messaging.outgoing.generated-price.apicurio.auth.service.url=${REGISTRY_IDP_URL}
+mp.messaging.outgoing.generated-price.apicurio.auth.realm=${REGISTRY_IDP_REALM}
+mp.messaging.outgoing.generated-price.apicurio.auth.client.id=${REGISTRY_CLIENT_ID}
```

Comment:
Question: Would be able to use the same service account for the service registry and kafka. I see that we have separate CLIENT_ID for kafka and REGISTRY_CLIENT_ID for registry?

Reply:
Good question.  Yes we can, and we should.  **However**, right now SR is only in staging, and I was using a production kafka cluster in my testing. So I needed separate SAs...

Reply:
I'd love us to make everything driven by convention for the simpler paths. That is use the same client/secret by default, but with the option to use separate ones if needed. 
Even use default SR and SSO URLs (for production env and the case when there is a single instance) so that the user doesn't have to specify it.

Reply:
We have a standard set of env var names for client secret and client id in quick starts - see for example https://github.com/redhat-developer/app-services-guides/tree/main/quarkus#configuring-the-quarkus-example-application-to-connect-to-a-kafka-instance

Reply:
Are we adding a quick start for this? I'm missing the text?